azure-pipelines.yml

```
@@ -315,10 +276,6 @@ jobs:
                                 "machine": "$(release.machine)",
                                 "end": "$(release.end)",
                                 "status": "$(release.status)"},
```

```suggestion
                                "status": "$(release.status)"}},
```
This is the last one in the list. Missing this will not fail CI but will cause issues when we (eventually) try to read the data.